features/notifications.yml.dist

```diff
@@ -1,4 +1,223 @@
 # Generated from: notifications.yml
 # Do not edit this file by hand. Edit the source file instead!
 
-{}
+status:
+  baseline: low
+  baseline_low_date: 2023-03-27
+  support:
+    chrome: "42"
+    chrome_android: "42"
+    edge: "17"
+    firefox: "44"
+    firefox_android: "44"
+    safari: "16"
+    safari_ios: "16.4"
+compat_features:
+  # baseline: low
+  # baseline_low_date: 2023-03-27
+  # support:
+  #   chrome: "40"
+  #   chrome_android: "40"
+  #   edge: "17"
+  #   firefox: "44"
+  #   firefox_android: "44"
+  #   safari: "16"
+  #   safari_ios: "16.4"
+  - api.ServiceWorkerRegistration.getNotifications
+
+  # ⬇️ Same status as overall feature ⬇️
+  # baseline: low
+  # baseline_low_date: 2023-03-27
+  # support:
+  #   chrome: "42"
+  #   chrome_android: "42"
+  #   edge: "17"
+  #   firefox: "44"
+  #   firefox_android: "44"
+  #   safari: "16"
+  #   safari_ios: "16.4"
+  - api.NotificationEvent
+  - api.NotificationEvent.NotificationEvent
+  - api.ServiceWorkerRegistration.showNotification
+
+  # baseline: false
+  # support:
+  #   chrome: "20"
+  #   edge: "14"
+  #   firefox: "22"
+  #   firefox_android: "22"
+  #   safari: "7"
+  #   safari_ios: "16.4"
+  - api.Notification
+  - api.Notification.Notification
+  - api.Notification.close
+
+  # baseline: false
+  # support:
+  #   chrome: "20"
+  #   edge: "14"
+  #   firefox: "22"
+  #   firefox_android: "22"
+  #   safari: "15"
+  #   safari_ios: "16.4"
+  - api.Notification.requestPermission_static
+
+  # baseline: false
+  # support:
+  #   chrome: "20"
+  #   edge: "14"
+  #   firefox: "26"
+  #   firefox_android: "26"
+  #   safari: "7"
+  #   safari_ios: "16.4"
+  - api.Notification.dir
+
+  # baseline: false
+  # support:
+  #   chrome: "32"
+  #   edge: "14"
+  #   firefox: "22"
+  #   firefox_android: "22"
+  #   safari: "7"
+  #   safari_ios: "16.4"
+  - api.Notification.permission_static
+
+  # baseline: false
+  # support:
+  #   chrome: "33"
+  #   edge: "14"
+  #   firefox: "26"
+  #   firefox_android: "26"
+  #   safari: "11"
+  #   safari_ios: "16.4"
+  - api.Notification.body
+  - api.Notification.icon
```

Comment:
Headsup that https://github.com/mdn/browser-compat-data/pull/24537 is going to make `icon` (and `tag`) be `false` on Safari.